src/components/Forms/Register.jsx

```diff
@@ -55,6 +55,7 @@ const Register = () => {
               maxLength={50}
               user={user}
               setUser={setUser}
+              editable={true}
```

Comment:
you can remove these lines now that the default is true. make sure to say its false in case u need that

Reply:
ah shoot sorry missed that one

Reply:
ok fixed register. All other forms alr fixed in prev commit